src/db/index.js

```diff
@@ -0,0 +1,54 @@
+class DB {
+    key;
+    schema;
+
+    constructor(key, schema) {
+        if (typeof key !== 'string' || typeof schema !== 'object' || key.length === 0 || schema.length === 0) {
+            return new Error('Bad parameter "key" or "schema"')
+        }
+        this.key = key
+        this.schema = schema
+    }
+
+    insert(entity) {
+        if (typeof entity !== 'object' || Object.keys(entity).length === 0 || Object.keys(entity).length !== this.schema.length) {
+            return new Error('Bad parameter "entity"')
+        }
+
+        if (!this.isSameKeys()) {
```

Comment:
The function `isSameKeys` takes a parameter `entity` which is not passed here.

Reply:
Thanks!